docs/operators/fixedWidthParserOperator.md

```diff
@@ -0,0 +1,240 @@
+| -------- | ----------- | ---- | ------------------ | ------------- |
+| *jsonSchema* | [Schema](https://github.com/apache/apex-malhar/blob/master/contrib/src/main/java/com/datatorrent/contrib/parser/FixedWidthSchema.java)  describing fixed width data. Based on schema definition the operator will parse the incoming record to object map and POJO. Valid records will be emitted as POJO / map while invalid ones are emitted on error port with error message. | String | YES | N/A |
+
+User can set this property in `properties.xml` in the following way:
+
+```xml
+<property>
+    <name>
+      dt.application.{ApplicationName}.operator.{OperatorName}.prop.jsonSchema
+    </name>
+    <value>
+    {
+      "padding": "_",
+      "alignment": "left",
+      "fields":
+      [
+        {
+          "name": "adId",
+          "type": "Integer",
+          "length": "3",
+          "padding": "0"
+        },
+        {
+          "name": "campaignId",
+          "type": "Integer",
+          "length": "3",
+          "padding": " "
+        },
+        {
+          "name": "adName",
+          "type": "String",
+          "length": "10",
+          "alignment":"right"
+        },
+        {
+          "name": "bidPrice",
+          "type": "Double",
+          "length": "3"
+        },
+        {
+          "name": "startDate",
+          "type": "Date",
+          "format": "yyyy-MM-dd HH:mm:ss",
+          "length": "19"
+        },
+        {
+          "name": "endDate",
+          "type": "Date",
+          "format": "dd/MM/yyyy",
+          "length": "10"
+        },
+        {
+          "name": "securityCode",
+          "type": "Long",
+          "length": "5"
+        },
+        {
+          "name": "active",
+          "type": "Boolean",
+          "length": "5",
+          "trueValue": "true",
+          "falseValue": "false"
+        },
+        {
+          "name": "optimized",
+          "type": "Boolean",
+          "length": "1",
+          "trueValue": "y",
+          "falseValue": "n"
+        },
+        {
+          "name": "parentCampaign",
+          "type": "String",
+          "length": "10"
+        },
+        {
+          "name": "weatherTargeted",
+          "type": "Character",
+          "length": "1"
+        }
+      ]
+    }
+    </value>
+</property>
+```
+Where {OperatorName} is the name of the Operator and {ApplicationName} is the name of the application.
+As explained earlier padding is a character used to specify padding used in the incoming records to fill the alloted maximum space if required. User has the flexibility to specify a single padding character for the entire file or the user can choose to provide separate padding character for separate fields (columns of the record). Similar to padding character user also has the flexibility to define the alignment of the incoming records, user can choose whether the record is left,centre or right aligned.
```

Comment:
There could only be left , right or centre alignment. The values are case insensitive hence not mentioning anything about the case. 
Have modified the text to take care of remaining comments